test/ui/digital-exchange/settings/common/Form.test.js

```diff
@@ -5,12 +5,32 @@ import { required, isNumber } from '@entando/utils';
 
 import RenderTextInput from 'ui/common/form/RenderTextInput';
 import SwitchRenderer from 'ui/common/form/SwitchRenderer';
-import { SettingsFormBody, maxLength50 } from 'ui/digital-exchange/settings/common/Form';
+import { SettingsFormBody as SettingsForm, maxLength50 } from 'ui/digital-exchange/settings/common/Form';
 
 const onWillMount = jest.fn();
 
+jest.unmock('react-redux');
+jest.unmock('redux-form');
+
+const mockIntl = {
```

Comment:
what about extracting it? it's used in lots of tests

Reply:
@gidesan  you mean `mockIntl` ?

Reply:
`jest.unmock('react-redux');
jest.unmock('redux-form');`
They are sometimes both used, sometimes only one of them, so extracting them would not make it better I think. As for `mockIntl` it should definitely be extracted.

Reply:
Yeah, I meant `mockIntl`.